addon/index.js

```diff
@@ -692,10 +693,10 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
      * @return {Array}
      */
     _rollbackKeys() {
-      return [
-        ...keys(get(this, CHANGES)),
-        ...keys(get(this, ERRORS))
-      ];
+      return uniq([
```

Comment:
You can use Ember's `uniq` here: http://devdocs.io/ember/classes/ember.enumerable/methods#uniq

So something like this:

```js
return emberArray([
  ...keys(get(this, CHANGES)), // don't need leafKeys anymore
  ...keys(get(this, ERRORS))   // don't need leafKeys anymore
]).uniq();
```